setup.cfg

```
@@ -0,0 +1,21 @@
+[metadata]
+name = py-gene-fusions
+description = Package for gene fusions
```

Maybe describe the functionality a little more?
```suggestion
description = Package for structuring gene fusions
```